### DISCUSSION TITLE
Simple LoRA Training GUI (ComfyUI exclusive)

### DISCUSSION THREAD

**Discussion by johnman3032** [comfyanonymous/ComfyUI#1791] [Show and tell]
https://github.com/johnman3032/simple-lora-dreambooth-trainer/

Stable Diffusion training is usually inaccessible to a lot of people due to the complexity of available training GUIs, this repo simplifies the process. It is built on the Dreambooth Diffusers scripts, and apparently only works with ComfyUI because other UIs load LoRAs differently.

**Comment by barrysutton** [comment]
Could someone point me to directions on how to use this LoRA training GUI in Google Colab? Thanks!
